code/modules/mob/mob.dm

```diff
@@ -634,6 +634,13 @@ var/list/slot_equipment_priority = list( \
 	if(!istype(W)) return 0
 
 	for(var/slot in slot_equipment_priority)
+		if(slot == slot_belt) //prioritize placing things in belt rather than swapping the belt slot
```

Comment:
Yeah, I probably should have made it a bit more generic.
I'll look into that then, thanks